configs/chrome/default.json

```diff
@@ -1,4 +1,5 @@
 {
+  "enable_create_new_profile": true,
```

Comment:
I think this should be applied to all config json files under `configs/chrome/` and `configs/firefox/`?